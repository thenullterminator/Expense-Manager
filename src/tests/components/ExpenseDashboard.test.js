import React from 'react';
import {shallow} from 'enzyme';
import ExpenseDashboard from '../../components/ExpenseDashboardPage';

test('should render expense dashboard page',()=>{

      const wrapper=shallow(<ExpenseDashboard />);
      expect(wrapper).toMatchSnapshot();

});