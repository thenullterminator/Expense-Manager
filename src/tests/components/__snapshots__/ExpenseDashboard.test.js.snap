// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense dashboard page 1`] = `
<div>
  <p>
    This is expense dashboard page.
  </p>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
